services/todo-legacy/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "Example Todo list application",
+  "main": "index",
+  "scripts": {
+    "start": "ts-node index.ts",
+    "test": "mocha --compilers ts:ts-node/register,tsx:ts-node/register test/**/*.test.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@loopback/testlab": "^4.0.0-alpha.2",
+    "@types/mocha": "^2.2.41",
+    "mocha": "^3.3.0",
+    "ts-node": "^3.0.4",
+    "typescript": "^2.3.2"
```

Comment:
Please move these two dependencies to `devDependencies` and also update them to the latest versions (TypeScript 2.4 was released recently).